src/image.rs

```diff
@@ -86,6 +86,7 @@ impl From<io::Error> for ImageError {
 pub type ImageResult<T> = Result<T, ImageError>;
 
 /// Result of a decoding process
+#[derive(Debug)]
```

Comment:
Actually I did use this but only in the PPM tests, so it is indeed strictly needed for this code.